pkgs/applications/misc/revelation/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, pkgconfig, python2Packages, gnome2, intltool }:
+
+stdenv.mkDerivation rec {
+  name = "revelation-${version}";
+  version = "0.4.14";
+
+  src = fetchurl {
+    url = https://bitbucket.org/erikg/revelation/downloads/revelation-0.4.14.tar.bz2;
+    sha256 = "2ab3d1d8bcc2f441feb58122ee6a0fe4070412228194843a180a7b1c9e910019";
+  };
+
+  patches = [ ./remove-randpool.patch ];
+
+  buildInputs = [ pkgconfig gnome2.GConf intltool ];
+
+  nativeBuildInputs = [
+    python2Packages.wrapPython
+  ];
+
+  propagatedBuildInputs = with python2Packages; [
+    pygtk pygobject2 pycrypto dbus-python gnome2.gnome_python python2Packages.cracklib
+  ];
+
+  preBuild = ''
+    export NIX_CFLAGS_COMPILE="$NIX_CFLAGS_COMPILE $(pkg-config --cflags gtk+-2.0) $(pkg-config --cflags pygtk-2.0)"
+  '';
+
+  postFixup = ''
+    wrapPythonPrograms
+  '';
+
+  meta = {
+    description = "A password manager for the GNOME desktop, released under the GNU GPL license";
```

Comment:
The "released under the GNU GPL license" is somehow redundant.

Reply:
I copied the description from the website, but you are right, it is redundant.